client/data/block-editor/use-has-seen-whats-new-modal-query.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery, UseQueryResult } from 'react-query';
+import wpcom from 'calypso/lib/wp';
+
+export type WhatsNewResult = {
+	has_seen_whats_new_modal: boolean;
+};
+
+export const useHasSeenWhatsNewModalQuery = (
```

Comment:
New query to call the new APIs from calypso

Reply:
I would put this one here: `packages/data-stores/src/queries`